src/universal-dapp.js

```diff
@@ -336,7 +421,7 @@ UniversalDApp.prototype.getInstanceInterface = function (contract, address, $tar
       var eventFilter = self.web3.eth.contract(abi).at(address).allEvents()
       eventFilter.watch(parseLogs)
     }
-    $instance.append($title)
+    $instance.get(0).appendChild(title)
```

Comment:
@chriseth This part is just jQuery refactoring
.append is a jQuery thingy and .appendChild works on dom elements. And therefore I change $instance into $instance.get(0) to get a dom element so I can use .appendChild on it. 

It's a slow process of removing jQuery, but I would rather do it bit by bit, otherwise we need few more days to refactor the whole thing now. I would suggest we leave it like this. I could do var instance = yo`<div>something</div>`, but then we use this $instance lots of times and this would cause a chain reaction, so... maybe we could skip it for now.